File: 백준/Silver/11399. ATM/ATM.cc
#include <iostream>
#include <algorithm>
using namespace std;

int a[1002];
int n,ret;
int main() {
	cin >> n;
	for (int i = 0; i < n; i++) {
		cin >> a[i];
	}
	sort(a,a+n);
	for (int i = 0; i < n; i++) {
		for (int j = 0; j <= i; j++) {
			ret += a[j];
		}
	}
	cout << ret << endl;
}

File: 백준/Silver/11399. ATM/ATM.java
import java.util.*;

public class Main {
    static int n, ret;
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        n = sc.nextInt();
        List<Integer> arr = new ArrayList<>();
        for (int i=0; i<n; i++){
            int num =0;
            num = sc.nextInt();
            arr.add(num);
        }
        Collections.sort(arr);
        for (int i=0; i<n; i++){
            ret += arr.get(i) * (n-i);
        }
        System.out.println(ret);
    }
}